app/views/devise/two_factor_authentication/show.html.slim

```diff
@@ -11,7 +11,7 @@ p
 
 = form_tag([:user, :two_factor_authentication], method: :put, role: 'form') do
   .mb2
-    = label_tag 'code', raw('<abbr title="required">*</abbr> ') + t('forms.two_factor.code')
```

Comment:
ahh, makes sense. we currently have a lot of other inputs that are required without this tag right now -- might if i do the form helper in a new PR and make sure it's applied across all views?